### PR TITLE
Translate "HTML" to Ukrainian

### PR DESCRIPTION
**Pull request by Herobread**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Oleksii Nazarenko

## Language: 

- Ukrainian

## Terms defined:

- html

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -5944,6 +5944,12 @@
     term: የሃይፐር ጽሑፍ ምልክት ማድረጊያ ቋንቋ
     def: >
       ለድር ገጾች ጥቅም ላይ የዋለው መደበኛው [ምልክት ማድረጊያ ቋንቋ](#markup_language) ኤችቲኤምኤል ነው [DOM](#dom) (ዲጂታል ነገር ሞዴል) በመጠቀም በማስታወስ ውስጥ ተወክሏል።
+  uk:
+    term: "мова розмітки гіпертексту (HyperText Markup Language)"
+    acronym: "HTML"
+    def: >
+      Стандартна [мова розмітки](#markup_language), що використовується для веб-сторінок. HTML
+      представлено у пам'яті за допомогою [DOM](#dom) (цифрова модель об'єктів).
```
Comment (olexandr-konovalov) [inline glossary.yml:5952]: DOM = Document_Object_Model, see e.g. https://en.wikipedia.org/wiki/Document_Object_Model - this should be corrected in the English version too and reviewed by someone who works on the English text. 

Note that German text says it right: Dokumenten-Objekt-Modell, as well as https://glosario.carpentries.org/en/#dom

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]

File: glossary.yml (hunk 1 above)
Comment (Herobread) [reply glossary.yml]: yes, should I create PR to fix English version?

File: glossary.yml [hunk 2]
```diff
@@ -5944,6 +5944,12 @@
     term: የሃይፐር ጽሑፍ ምልክት ማድረጊያ ቋንቋ
     def: >
       ለድር ገጾች ጥቅም ላይ የዋለው መደበኛው [ምልክት ማድረጊያ ቋንቋ](#markup_language) ኤችቲኤምኤል ነው [DOM](#dom) (ዲጂታል ነገር ሞዴል) በመጠቀም በማስታወስ ውስጥ ተወክሏል።
+  uk:
+    term: "мова розмітки гіпертексту (HyperText Markup Language)"
+    acronym: "HTML"
+    def: >
+      Стандартна [мова розмітки](#markup_language), що використовується для веб-сторінок. HTML
```
Comment (YehorBoiar) [inline glossary.yml:5951]: що використовується для створення та структурування веб-сторінок

Comment (Herobread) [reply glossary.yml]: @YehorBoiar updated, please review

File: glossary.yml (hunk 1 above)
Comment (olexandr-konovalov) [reply glossary.yml]: @Herobread - yes please, or alternatively, just add a separate commit which fixes it to this PR

File: glossary.yml (hunk 2 above)
Comment (olexandr-konovalov) [reply glossary.yml]: З 2019 року - вебсторінка без "-": див. https://www.uaredactor.com.ua/https-www-uaredactor-com-ua-zminy-v-novoy-redacciipravopysi/

File: glossary.yml [hunk 3]
```diff
@@ -5944,6 +5944,12 @@
     term: የሃይፐር ጽሑፍ ምልክት ማድረጊያ ቋንቋ
     def: >
       ለድር ገጾች ጥቅም ላይ የዋለው መደበኛው [ምልክት ማድረጊያ ቋንቋ](#markup_language) ኤችቲኤምኤል ነው [DOM](#dom) (ዲጂታል ነገር ሞዴል) በመጠቀም በማስታወስ ውስጥ ተወክሏል።
+  uk:
+    term: "мова розмітки гіпертексту (HyperText Markup Language)"
+    acronym: "HTML"
+    def: >
+      Стандартна [мова розмітки](#markup_language), що використовується для створення 
+      та структурування веб-сторінок. HTML представлено у пам'яті за допомогою [DOM](#dom) (об'єктна модель документа).
```
Comment (YehorBoiar) [inline glossary.yml:5952]: "веб-сторінок" -> "вебсторінок"

**Comment by Herobread** [comment]
@olexandr-konovalov @YehorBoiar updated according to your suggestions, please review

**Review by olexandr-konovalov — APPROVED** [review]

**Comment by YehorBoiar** [comment]
LGTM!

**Pull request merged**